### DISCUSSION TITLE
Transparent Bar, Keeping Text Solid/Opaque

### DISCUSSION THREAD

**Discussion by christopherbui** [qtile/qtile#2863] [Ideas] [answered]
The title says it all! It would be nice to have text on the bar remain opaque, with only the background be transparent. I believe this brings better legibility to the info displayed. 

I was wondering if this could be a possible feature, or would this depend on the compositor? I doubt the latter as the terminal Alacritty supports this opaque text with transparent background feature when using Picom.

I would appreciate any input, thanks!

**Comment by elParaguayo** [comment, marked as answer]
Did you try it? This is already available. You do need a compositor of using X11.

Just change the bar background colour so it includes an alpha value. 

Have a read of the docs: http://docs.qtile.org/en/latest/manual/config/screens.html?highlight=Transparency#example

**Reply by christopherbui** [reply]
Thank you! I had tried previously before and did not work. Must have been something wrong on my end then. Thank you for mentioning this!
